README.md

```diff
@@ -49,6 +49,7 @@ Some examples require extra dependencies. See each sample's directory for specif
     while running.
   * [hello_signal](hello/hello_signal.py) - Send signals to a workflow.
 <!-- Keep this list in alphabetical order -->
+* [activities_sticky_queues](activity_worker) - Uses unique task queues to ensure activities run on specific workers.
```

Comment:
```suggestion
* [activities_sticky_queues](activities_sticky_queues) - Uses unique task queues to ensure activities run on specific workers.
```

Also, though I know it doesn't match TS examples, I think `activity_sticky_queue` is a better sample/module name here.